### PR TITLE
Add bare minimum to allow connecting profiler to clio docker (#198)

### PR DESCRIPTION
**Pull request by jacarey**
Added clio-profile.xml for offline profiling

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

**Review by pshapiro4broad — APPROVED** [review]
Ugh, pile of XML

👍

File: project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala [hunk 1]
```diff
@@ -35,7 +37,17 @@ object Docker {
...
+        s"wget -q https://download-keycdn.ej-technologies.com/jprofiler/$JProfilerVersion.tar.gz -P /tmp/ && " +
+          s"tar -xzf /tmp/$JProfilerVersion.tar.gz -C /usr/local &&" +
+          s" rm /tmp/$JProfilerVersion.tar.gz"
+      )
+      env(
+        "JPAGENT_PATH",
+        "-agentpath:/usr/local/jprofiler9/bin/linux-x64/libjprofilerti.so=nowait"
+      )
+
+      expose(8080, 31757)
```
Comment (danxmoran) [inline project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: Wot wasn't this already merged? Maybe needs a rebase...

File: project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala [hunk 2]
```diff
@@ -35,7 +37,17 @@ object Docker {
...
       label("CLIO_VERSION", version.value)
-      expose(8080)
+      runRaw(
+        s"wget -q https://download-keycdn.ej-technologies.com/jprofiler/$JProfilerVersion.tar.gz -P /tmp/ && " +
+          s"tar -xzf /tmp/$JProfilerVersion.tar.gz -C /usr/local &&" +
+          s" rm /tmp/$JProfilerVersion.tar.gz"
+      )
+      env(
+        "JPAGENT_PATH",
+        "-agentpath:/usr/local/jprofiler9/bin/linux-x64/libjprofilerti.so=nowait"
```
Comment (danxmoran) [inline project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: Do we need any new env var to point at the conf file? Or will we need to specify it manually whenever we start up `jprofiler`?

Comment (jacarey) [reply project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: we specify it when we run jpenable in offline mode

jpenable -o -c /etc/clio-profile.xml

File: project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala (hunk 1 above)
Comment (jacarey) [reply project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala]: oh weird.. yeah

**Comment by jacarey** [comment]
yeah the jprofiler xml is ugly.. if we can get real time profiling set up we can remove it.

**Review by danxmoran — APPROVED** [review]

**Pull request merged**